blockchain/pallets/ddcclusters.go

```diff
@@ -26,15 +26,16 @@ type ClusterProps struct {
 
 type DdcClustersApi interface {
 	GetClustersNodes(clusterId ClusterId) ([]NodePubKey, error)
+	GetClusterInfo(clusterId ClusterId) (types.Option[Cluster], error)
```

Comment:
Let's rename to "GetClusters" to keep the getter name consistent with the storage item name `Clusters`.